sdks/python/apache_beam/examples/complete/tfidf.py

```diff
@@ -68,7 +68,8 @@ def expand(self, uri_to_content):
     # Create a collection of pairs mapping a URI to each of the words
     # in the document associated with that that URI.
 
-    def split_into_words((uri, line)):
+    def split_into_words(uri_line):
```

Comment:
Is this because of https://www.python.org/dev/peps/pep-3113/? This is sad.

Reply:
Yup